docs/Meadow/Meadow.OS/Cellular/index.md

```diff
@@ -19,31 +31,54 @@ Then, to configure your network you just need to follow these three steps:
 
 ```yaml
 Device:
     ReservedPins: I9;H13;C7
 ```
 
+> **Notes**: You can find the pins `I9`, `H13` and `C7` as `D00`, `D01` and `D10` on the **Meadow F7v2 Feather**, respectively.
+
 ## Hardware configuration
 
-To use the **BG770A Cell Wing with the F7 Feather v2**, you just need to attach them, and connect an *LTE* antenna to the cell module.
+To set up the hardware, the easiest way is to use our **BG770A Cell Wing** by attaching it to the **Meadow F7v2 Feather**. This provides a straightforward and hassle-free solution for establishing cellular connectivity. However, you can also use other cell click board models. All you need to do is connect the click boards to the **Meadow F7v2 Feather** using a few jumpers. To set up the hardware, follow the instructions based on your cell click board model:
+
+### Quectel BG770A - Meadow Cell Wing
+To utilize the **BG770A Cell Wing** with the **Meadow F7v2 Feather**, simply attach them, connect an LTE antenna (Rubber ducky or Dome) with an SMA Plug connector, aiming for a preferred gain of 5 dBi (recommended) while ensuring a minimum gain of 2 dBi (required), and insert an **M2M** SIM card into the cell module.
+
+### Quectel BG95-M3 - NimbeLink Skywire click board
+
+To use this module you will need to connect the **Meadow F7v2 Feather** `D00` and `D01` pins to the `TX` and `RX` click board pins, respectively, to establish the data communication between them.
+
+Also, you need to connect the **Meadow F7v2 Feather** `D10` pin to the `EN` **NimbeLink Skywire click board** pin. If you are using another click board for the **BG95-M3** module, you need to connect the `D10` pin to the equivalent power-up pin.
+
+Finally, connect an LTE antenna (Rubber ducky or Dome) with an SMA Plug connector, aiming for a preferred gain of 5 dBi (recommended) while ensuring a minimum gain of 2 dBi (required), and insert an **M2M** SIM card into the cell module.
+
+### Quectel M95 - GSM2 click board
+To use this module you will need to connect the **Meadow F7v2 Feather** `D00` and `D01` pins to the `TX` and `RX` click board pins, respectively, to establish the data communication between them.
+
+Also, you need to connect the **Meadow F7v2 Feather** `D10` pin to the `PWK` **Quectel GSM2 click board** pin, to turn on the module. If you are using another click board for the **M95** module, you need to connect the `D10` pin to the equivalent power-up pin.
+
+Finally, establish a connection by attaching a GSM antenna (Rubber ducky) with an SMA Plug connector, aiming for a preferred gain of 5 dBi (recommended) while ensuring a minimum gain of 2 dBi (required), and insert a SIM card into the cell module.
+
+> **Notes**: To enable **Cat-M1** (LTE-M or eMTC) or **NB-IoT** network modes, a specialized **M2M** (Machine-to-Machine) SIM card is required, distinct from the standard SIM cards used in cellphones. However, for **GSM/GPRS 2G** connections, a standard SIM card can generally be used.
 
 ## Testing
```

Comment:
Do you think that would be nice to have a FAQ section here? @alexischr 

We can try to add some of the common issues